service/config_provider.go

```diff
@@ -145,7 +145,7 @@ func (cm *configProvider) Watch() <-chan error {
 }
 
 func (cm *configProvider) onChange(event *configmapprovider.ChangeEvent) {
-	// TODO: Remove check for configsource.ErrSessionClosed when providers updated to not call onChange when closed.
+	// TODO: Remove check for configsource.ErrSessionClosed when providers updated to not call watcher when closed.
```

Comment:
should this have been changed? the method name here is still onChange

Reply:
GoLand decided to do this.